.github/workflows/deploy.yml

```diff
@@ -74,8 +74,8 @@ jobs:
           npx make deploy-lambdas-testnet
         env:
           API_PORT: 3001
-          HATHOR_CORE_DOMAIN: node1.foxtrot.testnet.hathor.network
-          HATHOR_NODES: node1.foxtrot.testnet.hathor.network
+          HATHOR_CORE_DOMAIN: node1.golf.testnet.hathor.network
+          HATHOR_NODES: node1.golf.testnet.hathor.network
```

Comment:
Maybe we should change to `node1.testnet.hathor.network`, so we don't need to update here every time we replace the testnet by a new one.

Reply:
Makes sense.